src/back/auth/src/api/helpers.cpp

```diff
@@ -2,22 +2,32 @@
 
 namespace svetit::auth::handlers {
 
-std::string getCallerUrl(const server::http::HttpRequest& req, bool addApiPrefix)
+std::string getCallerUrl(
+	const server::http::HttpRequest& req,
+	const formats::json::Value& params,
+	bool addApiPrefix)
 {
-	auto scheme = req.GetHeader("X-Forwarded-Proto");
-	if (scheme.empty())
+	std::string scheme;
+	if (!params.HasMember("X-Forwarded-Proto")) {
 		scheme = "http";
+	} else {
+		scheme = params["X-Forwarded-Proto"].As<std::string>();
+	}
```

Comment:
Не используем else там где можно без него.
```suggestion
	std::string scheme = "http";
	if (params.HasMember("X-Forwarded-Proto"))
		scheme = params["X-Forwarded-Proto"].As<std::string>();
```

Проверь пожалуйста, должно работать что-то такое:
```suggestion
	const auto scheme = params["X-Forwarded-Proto"].As<std::string>("http");
```
Если работает как ожидается, используй такое пожалуйста везде в затронутом коде.

Reply:
Готово, работает.